addons/binding/org.openhab.binding.dwdunwetter/README.md

```diff
@@ -0,0 +1,114 @@
+# DwdUnwetter Binding
+
+Binding to retrieve the Weather Warnings of the "Deutscher Wetterdienstes" from the [DWD Geoserver](https://maps.dwd.de/geoserver/web/ "DWD Geoserver").
+The DWD provides weather warnings for the german area. Regions outside of germany are not supported.
```

Comment:
Please capitalize country names in English:

```suggestion
The DWD provides weather warnings for the German area. Regions outside of Germany are not supported.
```

Reply:
Done